28_events/script.js

```diff
@@ -0,0 +1,34 @@
+// version 1
+
+let imgs = document.querySelector('.gallery-img');
+
+imgs.addEventListener('click', (event) => {
+    let img = event.target;
+    if (img.classList.contains('big-img')) {
+        img.classList.remove('big-img');   
+    } else {
+        img.classList.add('big-img');

```

Comment:
Thank you. I thought about it. But he didn't come up with anything quickly, and since there was no such condition in the assignment, he moved on to the next homework assignment.